product-owners.yml

```diff
@@ -0,0 +1,131 @@
+product_areas:
```

Comment:
Do we want our real PA/team structure here? I was just imagining a small test file with like 2 PAs, 2 teams, 1 additiona_user, etc? +1'ed the PR in any case.

Reply:
Yes, we do. We'll use this later on for other automations and it's not sensitive data so it's safe.